packages/server/package.json

```diff
@@ -3,56 +3,59 @@
   "version": "3.6.7",
   "description": "Core engine for Apollo GraphQL server",
   "type": "module",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
   "exports": {
     ".": {
+      "types": "./dist/esm/index.d.ts",
       "import": "./dist/esm/index.js",
-      "require": "./dist/cjs/index.js",
-      "types": "./dist/cjs/index.d.ts"
+      "require": "./dist/cjs/index.js"
     },
     "./standalone": {
+      "types": "./dist/esm/standalone/index.d.ts",
       "import": "./dist/esm/standalone/index.js",
-      "require": "./dist/cjs/standalone/index.js",
-      "types": "./dist/cjs/standalone/index.d.ts"
+      "require": "./dist/cjs/standalone/index.js"
     },
     "./plugin/cacheControl": {
+      "types": "./dist/esm/plugin/cacheControl/index.d.ts",
       "import": "./dist/esm/plugin/cacheControl/index.js",
-      "require": "./dist/cjs/plugin/cacheControl/index.js",
-      "types": "./dist/cjs/plugin/cacheControl/index.d.ts"
+      "require": "./dist/cjs/plugin/cacheControl/index.js"
     },
     "./plugin/disabled": {
+      "types": "./dist/esm/plugin/disabled.d.ts",
```

Comment:
Wanna make the index change here for consistency?

Reply:
...and maybe we need to smoke test these paths since presumably this is in a broken state without CI knowing.